integration-tests/esbuild/index.spec.js

```diff
@@ -15,19 +15,15 @@ const TEST_DIR = path.join(__dirname, '.')
 process.chdir(TEST_DIR)
 
 // This should switch to our withVersion helper. The order here currently matters.
-const esbuildVersions = ['latest', '0.16.12']
+const latestVersion = require('../../packages/dd-trace/test/plugins/versions/package.json').dependencies.esbuild
+const esbuildVersions = [latestVersion, '0.16.12']
+
+const timeout = 60_000
 
 esbuildVersions.forEach((version) => {
   describe(`esbuild ${version}`, () => {
     before(() => {
-      chproc.execSync('npm install', {
-        timeout: 1000 * 30
-      })
```

Comment:
Why did we run this `npm install` before?

Reply:
It was just a hack for the setup for now. We should instead use our withVersions helper instead. There is already a comment about that at the top.